src/main/k8s/assets/adot_collector/deployment.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: adot-collector
+  namespace: ${namespace}
+  labels:
+    app: aws-adot
+    component: adot-collector
+spec:
+  selector:
+    matchLabels:
+      app: aws-adot
+      component: adot-collector
+  template:
+    metadata:
+      labels:
+        app: aws-adot
+        component: adot-collector
+    spec:
+      serviceAccountName: adot-collector
+      securityContext:
+        fsGroup: 65534
+      containers:
+        - image: public.ecr.aws/aws-observability/aws-otel-collector:latest
```

Comment:
The image version must be pinned! If they release a breaking change we could lose metrics. Use a TF variable and parametrize

Reply:
done